src/app/resources/iis_scripts/admin_api_util.ps1

```diff
@@ -95,25 +166,34 @@ if (!$config.security.users.owners.Contains($user)) {
         $dirAcl = Get-Acl $apiHome
         foreach ($access in $dirAcl.Access | Where-Object { $_.IdentityReference.Value -eq $user }) {
             $dirAcl.RemoveAccessRule($access) | Out-Null
+            $updateAcl = $true
+        }
+        if ($updateAcl) {
+            Write-Verbose "Updating Acl to remove readonly access..."
+            Set-Acl -Path $apiHome -AclObject $dirAcl | Out-Null
         }
-        Set-Acl -Path $apiHome -AclObject $dirAcl | Out-Null
         $rights = [System.Security.AccessControl.FileSystemRights]::Traverse `
             -bOr [System.Security.AccessControl.FileSystemRights]::Modify `
             -bOr [System.Security.AccessControl.FileSystemRights]::ChangePermissions
-        $dirAccessGranted = $dirAcl.Access | Where-Object {(($_.IdentityReference.Value -eq "${env:ComputerName}\${iisAdminOwners}") -and (($_.FileSystemRights -bAnd $rights) -eq $rights))}
+        $dirAccessGranted = $dirAcl.Access | Where-Object {(($_.IdentityReference.Value -eq "BUILTIN\Administrators") -and (($_.FileSystemRights -bAnd $rights) -eq $rights))}
         if (!$dirAccessGranted) {
             $inheritFlags = [System.Security.AccessControl.InheritanceFlags]::ContainerInherit `
                 -bOr [System.Security.AccessControl.InheritanceFlags]::ObjectInherit
             $propFlags = [System.Security.AccessControl.PropagationFlags]::InheritOnly
             $ar = New-Object System.Security.AccessControl.FileSystemAccessRule("BUILTIN\Administrators", $rights, $inheritFlags, $propFlags, "allow")
             $dirAcl.SetAccessRule($ar) | Out-Null
+            Write-Verbose "Setting Acl to enable administrators' write access"
             Set-Acl -Path $apiHome -AclObject $dirAcl | Out-Null
         }
-        $config.security.users.owners += $iisAdminOwners
+        Write-Verbose "Saving new configuration"
         $config | ConvertTo-Json -depth 100 | Out-File $configLocation

```

Comment:
After file is updated, please check if the updated configuration is correctly updated for double-check. If the configuration was not updated correctly, we should show error message here.

Reply:
If something went wrong Out-File would have thrown an error. It's unnecessary to check here. It's up to powershell to ensure that this works